hack/konveyor-analyzer

```diff
@@ -16,7 +16,7 @@ do
 done
 
 docker run \
-  --entrypoint /usr/local/bin/konveyor-analyzer \
+  --entrypoint konveyor-analyzer \
```

Comment:
does this need to be `/usr/bin/konveyor-analyzer`?  Assume it doesn't matter since its in PATH

Reply:
This is a hack I created to run the analyzer locally as a command. I don't know if this will work without the path or not. @shawn-hurley made the changes.

Reply:
we will see when CI runs